src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get "
+      class="bar1"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        placeholder="Try &quot;los Angeles&quot;"
+        class="input1"
```

Comment:
you repeat your styles, what you can do to avoid doing that
```
...
<input class="input input--big">
...
<input class="input input--small">
```
`.input` – common styles
`.input--big` – styles for big input only
`.input--small` – styles for small input only